CoreGraphNode/src/au/gov/asd/tac/constellation/graph/node/plugins/DefaultPluginEnvironment.java

```diff
@@ -97,7 +100,8 @@
 
             try {
                 ConstellationLogger.getDefault().pluginStarted(plugin, parameters, graph);
-            } catch (Exception ex) {
+            } catch (final Exception ex) {
+                LOGGER.log(Level.WARNING, EXCEPTION_MESSAGE, ex);
```

Comment:
Should've mentioned it in my earlier comment but passing the exception as an argument will actually halt the execution (from previous experience) so just omit that so that execution runs as it did. I would also change the warning level to Severe since technically getting here means something went wrong:
`LOGGER.log(Level.SEVERE, EXCEPTION_MESSAGE)`

Similarly for the other log entries you added

Reply:
So just log it with a default message like "An unexpected exception was caught."
 ?

Reply:
For now I would maybe log `ex.getLocalisedMessage()`

Reply:
Done, let me know if that's how you meant

Reply:
Yep that's what I meant. The only thing I would add is change the log level to Severe rather than Warning. While we're not throwing the exception, getting to those catches does technically mean something went wrong so best to log it as much

Reply:
Updated